### PR TITLE
Multifloor navigator

### PR DESCRIPTION
**Pull request by owen7900**
This includes base functionality for mutlifloor planning through a pose graph provided by map_nodes.yaml. Tools to create, view and edit those pose graph files will need to be made.

### CODE REVIEW

File: master_navigator/include/master_navigator/MasterNavigator.hpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#pragma once
+
```
Comment (amaclatchy) [inline master_navigator/include/master_navigator/MasterNavigator.hpp:2]: We should decide on a naming convention for private member variables to help my small brain. I've always used the leading underscore but I've seen most ROS stuff uses the trailing underscore. I don't really care which convention we use but I find it a lot easier to read

File: notes/interfaceDef.txt [hunk 2]
```diff
@@ -1,17 +1,19 @@
 GLOBAL
-CurrentFloor String(floor_id) Topic - pub by nav
+CurrentFloor String(floor_id) Topic - pub by nav --pubbed
```
Comment (amaclatchy) [inline notes/interfaceDef.txt:2]: Don't know what --pubbed means...
Same with the "pubbed" and "subbed" and "called" additions to line 8, 14, and 15

Comment (owen7900) [reply notes/interfaceDef.txt:2]: Just notes for myself to make sure I implemented everything I needd

File: master_navigator/include/master_navigator/MasterNavigator.hpp (hunk 1 above)
Comment (owen7900) [reply master_navigator/include/master_navigator/MasterNavigator.hpp:2]: We need to create a naming convention for things overall. We can chat tmr

Comment (owen7900) [reply master_navigator/include/master_navigator/MasterNavigator.hpp:2]: I'll leave this open until we determine what we are going to do.

**Pull request merged**